chugbot/attendance/chugLeaderHome.php

```diff
@@ -0,0 +1,169 @@
+<div class="container row justify-content-center" style="margin:auto;"><div id="errors" class="mt-2"></div></div>
+
+<div class="card card-body mt-2 p-3 mb-3 container">
+    <h1><?php echo ucfirst(chug_term_singular)?> Leader Home</h1>
+    <div class="page-header"><h2><?php echo ucfirst(chug_term_singular)?> Leader Home</h2>
+    <p>In the form below, select the date, edah/edot, perek, and <?php echo chug_term_singular?> you are taking attendance for.
+    After clicking "Take Attendance" below the form, check the box for each camper who is <strong>present</strong>. Then press "Submit attendance" to complete the attendance.</p>
+    
+    <form id="attendance_chug_select_form" class="justify-content-center" method="GET" action="takeAttendance.php" onsubmit="return validateForm()"><ul>
+        <li style="margin:auto;" class="ps-0">
+            <label class="description" for="date"><span style="color:red;">*</span>Date</label>
+            <div id="date_pick" class="pb-2">
+                <input type="date" id="date" name="date" class="form-control medium" required>
+            </div>
+        </li>
+        <li style="margin:auto;" class="ps-0">
+            <label class="description" for="edah[]"><span style="color:red;">*</span>Edah/Edot</label>
+            <div id="edah_select" class="pb-2">
+                <select class="form-select bg-info choices-js" id="edah_list" name="edah[]" onchange="fillConstraintsPickList(); fillChugimConstraintsPickList();" multiple>
+                    <?php echo genPickList($edahId2Name, array(), "edah"); ?>
+                </select>
+            </div>
+        </li>
+        <li style="margin:auto;" class="ps-0"> 
+            <label class="description" for="group" id="group_desc"><span style="color:red;">*</span>Perek</label>
+            <div id="group_select" class="pb-2">
+                <?php echo genConstrainedPickListScript("group_select", "edah", "group_desc", "group", true); ?>
+            </div>
+        </li>
+        <li style="margin:auto;" class="ps-0">
+            <label class="description" for="chug" id="chug_desc"><span style="color:red;">*</span><?php echo ucfirst(chug_term_singular) ?></label>
+            <div id="chug_select" class="pb-2">
+                <?php echo genChugimPickListScript("chug_select", "edah", "group", "chug_desc", "chug", true); ?>
+            </div>
+        </li>
+        <li style="margin:auto;">
+            <div class="row justify-content-center">
+                <div class="col-6" style="text-align:center;"><button class="btn btn-primary" type="submit" id="submit_btn">Take Attendance</button></div>
+            </div>
+        </li>
+    </ul></form>
+
+</div>
+
+<script>
+// automatically set date to "today"
+const today = new Date();
+const yyyy = today.getFullYear();
+const mm = String(today.getMonth() + 1).padStart(2, '0'); // Months start at 0!
+const dd = String(today.getDate()).padStart(2, '0');
+
+const formattedToday = `${yyyy}-${mm}-${dd}`;
```

Comment:
I was trying similar to that, but ran into problems in the evening where it would skip to the next day (my guess is based on UTC day/time instead of local time)... this Frankenstein-esque way just seemed to be more consistent

Reply:
Ah, got it! Yup, that makes sense. FWIW, I think `date` would let you localize by time zone, but it's fine to keep this simple. I ran into a similar issue when attempting to "guess" the current camp year: the current approach is to use the current year until September and then flip to the next year (e.g., right now the "current" summer is 2024, and in September it will roll over to 2025). Total seat-of-the-pants coding there. :)